Assets/MixedRealityToolkit/Utilities/Editor/EditorProjectUtilities.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.MixedReality.Toolkit.Core.Utilities.Editor.Setup;
 using UnityEditor;
 
 namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Editor
 {
     [InitializeOnLoad]
-    internal class EditorProjectUtilities
+    public class EditorProjectUtilities
```

Comment:
Does this need to be made public? I feel like most of the scripts that need to use these calls can stay in the core assembly, but I might be overlooking some use cases.

Reply:
the compiler balked when I left it as internal, presumably because other assemblies are using a public property that calls into this class.

Reply:
i can also envision instances where assemblies may wish to know relative folder paths